parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -135,6 +137,9 @@ pub mod pallet {
 
 		/// The upper limit here only used to estimate delivery cost
 		type MaxMessageSize: Get<u32>;
+
+		/// Something to execute an XCM message.
+		type XcmExecutor: ExecuteXcm<Self::RuntimeCall> + XcmAssetTransfers;
```

Comment:
Lets narrow the scope, and just rely on the transactor:

```suggestion
		type AssetTransactor: TransactAsset;
```